.vscode/settings.json

```diff
@@ -4,88 +4,91 @@
 	"cmake.parallelJobs": 16,
 	"cmake.preferredGenerators": [ "Ninja" ],
 	"cmake.buildArgs": [ "-v" ],
-	"cmake.configureOnEdit": true,
+	"cmake.configureOnEdit": false,
```

Comment:
Please, Ruslan, don't do `auto configure` on me...
It's extremely painful and I hate it.